x-pack/plugin/blob-cache/src/test/java/org/elasticsearch/blobcache/BlobCacheUtilsTests.java

```diff
@@ -25,6 +27,56 @@ public void testReadSafeThrows() {
     public void testToPageAlignedSize() {
         long value = randomLongBetween(0, Long.MAX_VALUE - SharedBytes.PAGE_SIZE);
         long expected = ((value - 1) / SharedBytes.PAGE_SIZE + 1) * SharedBytes.PAGE_SIZE;
-        assertThat(BlobCacheUtils.toPageAlignedSize(value), Matchers.equalTo(expected));
+        assertThat(BlobCacheUtils.toPageAlignedSize(value), equalTo(expected));
+        assertThat(BlobCacheUtils.toPageAlignedSize(value), equalTo(roundUpUsingRemainder(value, SharedBytes.PAGE_SIZE)));
+    }
+
+    public void testRoundUpToAlignment() {
+        assertThat(BlobCacheUtils.roundUpToAlignedSize(8, 4), equalTo(8L));
+        assertThat(BlobCacheUtils.roundUpToAlignedSize(9, 4), equalTo(12L));
```

Comment:
nit It'd be nice to also add an assertion like
`assertThat(BlobCacheUtils.roundUpToAlignedSize(2, 4), equalTo(4L));`

Reply:
I wonder why you find that one in particular important, it seems similar enough to the one you highlighted?

Reply:
In `testRoundDownToAlignment`, the first line tests a size < alignment, and I thought it'd be nice to have a similar test here as well. But it's a nit, so feel free to ignore if you do not find it useful/important.

Reply:
Can you point me to that line, I am not sure I see it?

Reply:
But let me add it in both places.

Reply:
Done in 128e73f190449c7a0d3eb0a3da88c787cd2cb3ca

Reply:
> Can you point me to that line, I am not sure I see it?

Huh, checking it again, you're right, I was mistaken! Probably I was thinking it would be nice to have size < alignment when I was reading `testRoundDownToAlignment` and made the comment.